xbmc/cores/VideoPlayer/VideoPlayerVideo.cpp

```diff
@@ -900,6 +902,9 @@ int CVideoPlayerVideo::OutputPicture(const DVDVideoPicture* src, double pts)
     return EOS_DROPPED;
   }
 
+  m_processInfo.SetVideoDimensions(pPicture->iWidth, pPicture->iHeight);
```

Comment:
VideoDimensions refer to video surface dimensions (uncropped). Should be set by decoder.